pilot/lib/pilot/options.rb

```diff
@@ -68,12 +68,14 @@ def self.available_options
                                      description: "Skip the distributing action of pilot and only upload the ipa file",
                                      is_string: false,
                                      default_value: false),
+        # rubocop:disable Metrics/LineLength
         FastlaneCore::ConfigItem.new(key: :skip_waiting_for_build_processing,
                                      short_option: "-z",
                                      env_name: "PILOT_SKIP_WAITING_FOR_BUILD_PROCESSING",
-                                     description: "Don't wait for the build to process. If set to true, the changelog won't be set, `distribute_external` option won't work and no build will be distributed to testers",
+                                     description: "Don't wait for the build to process. If set to true, the changelog won't be set, `distribute_external` option won't work and no build will be distributed to testers. \n Uploading and verifying submissions to TestFlight can take sometime, it may use up unexpected minutes in your allotted CI plans. Use skip_waiting_for_build_processing: true to reduce used minutes.",
```

Comment:
```
[14:57:59]: ▸ /home/circleci/project/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!': Do not let descriptions end with a '.', since it's used for user inputs as well for key :skip_waiting_for_build_processing (FastlaneCore::Interface::FastlaneError)
```` 

To fix I guess:

```suggestion
                                     description: "Don't wait for the build to process. If set to true, the changelog won't be set, `distribute_external` option won't work and no build will be distributed to testers. \n Uploading and verifying submissions to TestFlight can take sometime, it may use up unexpected minutes in your allotted CI plans. Use skip_waiting_for_build_processing: true to reduce used minutes",
```